examples/classification/client.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from typing import List, Union
+
+import numpy
+import requests
+
+from deepsparse.utils import arrays_to_bytes, bytes_to_arrays
+from examples.classification.helper import _BatchLoader, load_data
+
+
+class Client:
+    """
+    Client object for making requests to the example DeepSparse inference server
+    :param address: IP address of the server, default is 0.0.0.0
```

Comment:
nit: add blank line between main info and params

Reply:
🆗